openstack/baremetal/v1/nodes/urls.go

```diff
@@ -0,0 +1,59 @@
+package nodes
```

Comment:
Only include the URLs being used in this PR.

Reply:
Removed unused URLs